src/main/java/com/rapid7/jenkinspider/PostBuildScan.java

```diff
@@ -429,6 +430,7 @@ public FormValidation doTestCredentials(@QueryParameter("appSpiderEntUrl") final
         public FormValidation doValidateNewScanConfig(@QueryParameter("scanConfigName") final String scanConfigName,
                                                       @QueryParameter("scanConfigUrl") final String scanConfigUrl) {
             try {
+                String ALPHANUMERIC_REGEX = "^[a-zA-Z0_\\-\\.]*$";
```

Comment:
This is better as a constant (static final at top of class)

Reply:
I'd agree that it should be final, IDE or one of the analysis addons recommended moving it here which I'm inclined to agree with since it's only used here